invenio_iiif/handlers.py

```diff
@@ -28,12 +28,12 @@
 def image_opener(key):
     """Handler to locate file based on key.
 
-    :param key: A key encoded in the format "<bucket>e<object_key>:<version>".
+    :param key: A key encoded in the format "<bucket>:<version>:<object_key>".
     :returns: A file-like object.
     """
-    # Drop the "version" that comes after the second ":" - we use this version
+    # Drop the "version" that comes after the first ":" - we use this version
     # only as key in redis cache
-    bucket, object_key = key.split(':')[:2]
+    bucket, version, object_key = key.split(':')
```

Comment:
``key.split(':', 2)`` so that in case ``object_key`` contains colon, it will still do the correct job.